### DISCUSSION TITLE
why is minio running in Kubernetes as a single pod is 6 times slower then the running the minio server direct

### DISCUSSION THREAD

**Discussion by aviscode** [minio/minio#13037] [Q&A] [answered]
why is minio running in Kubernetes as a single pod is 6 times slower then running the minio server direct
as u can see 

Kubernetes :: avi@avi-KVM:/dev/shm$ mc cp test.img my/avi/
test.img:                5.00 GiB / 5.00 GiB ┃▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓┃ 149.16 MiB/s 34s

service :: avi@avi-KVM:/dev/shm$ mc cp test.img myminio/avi/
test.img:                5.00 GiB / 5.00 GiB ┃▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓┃ 950.46 MiB/s 5s

**Comment by aviscode** [comment]
@harshavardhana

**Comment by ravindk89** [comment, marked as answer]
@aviscode there is a lot of missing information here. We cannot even begin to diagnose a performance issue with only the commands above.

To start:

- What MinIO versions are you running?
- What do you mean by "Direct" - do you mean baremetal vs kubernetes?
- What hardware configuration are you using for baremetal vs kubernetes?
- How are you deploying MinIO onto Kubernetes?
- By single pod do you mean a standalone MinIO server? What startup command are you using

From there we will respond as we can - there are no SLA/SLO on these discussion boards, and at-mention/tags will not raise priority of an issue.
